bridger/display/metadata_config.py

```diff
@@ -19,7 +19,6 @@ def get_preview_display(self) -> Optional[InstanceDisplay]:
 
     def get_metadata(self):
         yield "instance", list(self.get_instance_display() or [])
-        yield "preview", list(self.get_preview_display() or [])
```

Comment:
Why are you removing this line?

Reply:
Because Christoph said this is not needed anymore (that the controller does not handle preview in display anymore)  See issue #104

Reply:
One of them can be removed, I am just unsure which one. If you remove this, does the preview still work on the bridger test server?

Reply:
Yes it does:
![image](https://user-images.githubusercontent.com/7704509/129020837-5901de67-457b-4901-97bd-b07e0ebdd88a.png)